zk.js/src/test-utils/test-rpc.ts

```diff
@@ -18,7 +18,12 @@ import { bs58 } from "@coral-xyz/anchor/dist/cjs/utils/bytes";
 import { LightWasm } from "@lightprotocol/account.rs";
 
 import { Rpc } from "../rpc";
-import { fetchRecentTransactions } from "../transaction";
+import {
+  ParsingUtxoBeet,
+  PublicTransactionIndexerEventBeet,
+  fetchRecentPublicTransactions,
+  fetchRecentTransactions,
```

Comment:
tbh might make sense to call both explicitly, i.e. fetchRecentPublicTransactions and fetchRecentPrivateTransactions/Encrypted